app/services/virtual_hearing_user_alert_builder.rb

```diff
@@ -13,27 +14,58 @@ def call
 
   private
 
-  attr_reader :change_type, :alert_type, :hearing
+  attr_reader :change_type, :alert_type, :hearing, :virtual_hearing_updates
 
   def title
     copy["TITLE"] % (hearing.appeal.veteran&.name || "the veteran")
   end
 
+  def appellant_title
+    hearing.appeal.appellant_is_not_veteran ? "Appellant" : "Veteran"
+  end
+
+  # Appellant is a recipient if the `appellant_email_sent` flag is changing.
+  def appellant_is_recipient?
+    !virtual_hearing_updates.fetch(:appellant_email_sent, true)
```

Comment:
```suggestion
    virtual_hearing_updates.key?(:appellant_email_sent)
```
I think this might be a more succinct way of checking this, since [the key will only be included in the hash if its value is `false`](https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/models/hearings/forms/base_hearing_update_form.rb#L198). Unless you want to future-proof for the possibility that we'll want to include other values at some point.